public/javascripts/SVLabel/css/svl.css

```diff
@@ -8,10 +8,10 @@
 text {
     visibility: hidden;
 }
-
+/*
```

Comment:
I would say leave it in there. You actually don't need the toolUI class. You could use addClass to add `.visible` to `#container` in line 393 of Main.js instead of having both lines 393 and 394.